controllers/metallb_controller.go

```diff
@@ -53,7 +54,7 @@ var ManifestPath = "./bindata/deployment"
 
 func (r *MetallbReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	_ = context.Background()
-	_ = r.Log.WithValues("metallb", req.NamespacedName)
+	logger := r.Log.WithValues("metallb", req.NamespacedName)
```

Comment:
Can you add "metallb reconcile" here so we know where we are logging?

Reply:
fixed